pkg/repository/udmrepo/kopialib/backend/common.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package backend
+
+import (
+	"context"
+	"time"
+
+	"github.com/kopia/kopia/repo"
+	"github.com/kopia/kopia/repo/blob"
+	"github.com/kopia/kopia/repo/blob/throttling"
+	"github.com/kopia/kopia/repo/content"
+	"github.com/kopia/kopia/repo/encryption"
+	"github.com/kopia/kopia/repo/hashing"
+	"github.com/kopia/kopia/repo/object"
+	"github.com/kopia/kopia/repo/splitter"
+
+	"github.com/vmware-tanzu/velero/pkg/repository/udmrepo"
+)
+
+const (
+	maxDataCacheMB         = 2000
+	maxMetadataCacheMB     = 2000
+	maxCacheDurationSecond = 30
+)
+
+func setupLimits(ctx context.Context, flags map[string]string) throttling.Limits {
+	return throttling.Limits{
```

Comment:
Actually, do we set all configurations into 0? 

and if setting it to 0, Dose kopia will not do any limitation?

Reply:
Yes, Kopia treats it as unlimited